cmd/root.go

```diff
@@ -40,6 +41,11 @@ func Execute() {
 	pf.StringVarP(&flagConfigFile, "config-file", "", "", "set path of configuration file")
 	config.KubeConfigFlags(pf, argToEnv)
 	config.LoggerFlags(pf, argToEnv)
+
+	pf.IntP("verbosity", "", 0, "Only print progress messages at or above this level")
```

Comment:
I think that we should provide some information what levels exist.

Reply:
The levels are numeric, zero and up. Currently only 0 and 1 used. Can try to place this in the help.

Reply:
Done with commit 6f842b54a673a184d1d728be9f37a580d056cdaa